code/modules/mining/mine_items.dm

```diff
@@ -249,4 +249,50 @@
 	if(colorindex >= 6)
 		colorindex = 0
 	icon_state = "mobcap[colorindex]"
-	update_icon()
\ No newline at end of file
+	update_icon()
+
+//Fans
+/obj/structure/fans
+	icon = 'icons/obj/lavaland/survival_pod.dmi'
+	icon_state = "fans"
+	name = "environmental regulation system"
+	desc = "A large machine releasing a constant gust of air."
+	anchored = 1
+	density = 1
+	var/arbitraryatmosblockingvar = 1
+	var/buildstacktype = /obj/item/stack/sheet/metal
+	var/buildstackamount = 5
+
+/obj/structure/fans/proc/deconstruct()
+	if(buildstacktype)
+		new buildstacktype(loc, buildstackamount)
+	qdel(src)
+
+/obj/structure/fans/attackby(obj/item/weapon/W, mob/user, params)
+	if(istype(W, /obj/item/weapon/wrench))
+		playsound(src.loc, 'sound/items/Ratchet.ogg', 50, 1)
+		user.visible_message("<span class='warning'>[user] disassembles the fan.</span>", \
+							 "<span class='notice'>You start to disassemble the fan...</span>", "You hear clanking and banging noises.")
+		if(do_after(user, 20, target = src))
+			deconstruct()
```

Comment:
Yeah, I don't want players making these because of how powerful they are (look ma, I'm in a room in the middle of space with no walls that has breathable air!)